File: Java/1133. Largest Unique Number.java
class Solution {
    public int largestUniqueNumber(int[] nums) {
        int[] bucket = new int[1001];
        for (int n: nums) {
            bucket[n]++;
        }

        for (int i = 1000; i >= 0; i--) {
            if (bucket[i] == 1) 
                return i;
            
        }

        return -1;
    }
}

File: Python3/1133. Largest Unique Number.py
class Solution:
    def largestUniqueNumber(self, nums: List[int]) -> int:
        bucket = [0] * 1001
        for n in nums:
            bucket[n] += 1

        for i in range(1000, -1, -1):
            if bucket[i] == 1:
                return i

        return -1